src/internal/kopia/conn.go

```diff
@@ -38,6 +39,24 @@ const (
 var (
 	ErrSettingDefaultConfig = clues.New("setting default repo config values")
 	ErrorRepoAlreadyExists  = clues.New("repo already exists")
+
+	// minEpochDurationLowerBound is the minimum corso will allow the kopia epoch
+	// duration to be set to. This number can still be tuned further, right now
+	// it's just to make sure it's not set to something totally wild.
+	//
+	// Note that there are still other parameters kopia checks when deciding if
+	// the epoch should be changed. This is just the min amount of time since the
+	// last epoch change that's required.
+	minEpochDurationLowerBound = 2 * time.Hour
```

Comment:
question: what's an epoch / what does this configure?

Reply:
this is for kopia's epoch manager which holds the index mapping content IDs -> (S3 blob, blob offset). The epoch manager is kind of like a log structured merge tree, it keeps the most recent epochs "uncompacted" with lots of little files denoting the changes in the epoch (additions/removals of content). All files corresponding to an epoch need to be loaded to create a complete view of the index and access kopia's data

Once the epoch is old enough, it gets compacted into a summary of sorts. The summary is a single S3 blob that contains all the index changes for one or more epochs (kopia compacts compacted epochs to further reduce the number of S3 GETs)

This parameter denotes the minimal amount of time that has to pass before switching to a new epoch. By changing this, one can tune how many S3 blobs may need to be fetched in order to access kopia's data. This works by reducing the number of possible blobs that can accrue in an epoch prior to switching to a new epoch (and thus compacting an older one) by letting kopia switch epochs more often